### PR TITLE
Create the-years-that-the-days-and-months-turned-into_shafer-hall.md

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: "The Years That The Days and Months Turned Into"
+authors:
+  - "Shafer Hall"
+subcat: poetry
```
Comment (buddhist-uni) [inline _content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md:5]: add a field:

```yaml
reader: "Ada Limón"
```

File: _content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+external_url: "https://www.slowdownshow.org/episode/2022/05/23/680-the-years-that-the-days-and-months-turned-into"
+source_url: ""
```
Comment (buddhist-uni) [inline _content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md:7]: remove empty fields

File: _content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  - contemporary-poetry
+year: 2022
+month: may
+publisher: "American Public Media"
+series: slowdown
+number: 680
+minutes: 5
+---
+
+> I was the angry one, and I was the sad one,
```
Comment (buddhist-uni) [inline _content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md:23]: Subtle, but in markdown, lines of poetry must end in two spaces to mark the line break as intentional.

File: _content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+subcat: poetry
+external_url: "https://www.slowdownshow.org/episode/2022/05/23/680-the-years-that-the-days-and-months-turned-into"
+source_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/1L1cUFxeqCq2FTsAsVSCSmbrHGTrHo9dW/view?usp=sharing"
+formats: [mp3]
+course: thought
+tags:
+  - imagination
+  - contemporary-poetry
```
Comment (buddhist-uni) [inline _content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md:14]: Or even just `poetry` as I think it's pretty accessible

File: _content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+authors:
+  - "Shafer Hall"
+subcat: poetry
+external_url: "https://www.slowdownshow.org/episode/2022/05/23/680-the-years-that-the-days-and-months-turned-into"
+source_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/1L1cUFxeqCq2FTsAsVSCSmbrHGTrHo9dW/view?usp=sharing"
+formats: [mp3]
+course: thought
+tags:
```
Comment (buddhist-uni) [inline _content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md:12]: Let's add `wider` as a tag here 😁

File: _content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+title: "The Years That The Days and Months Turned Into"
+authors:
+  - "Shafer Hall"
+subcat: poetry
+external_url: "https://www.slowdownshow.org/episode/2022/05/23/680-the-years-that-the-days-and-months-turned-into"
+source_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/1L1cUFxeqCq2FTsAsVSCSmbrHGTrHo9dW/view?usp=sharing"
+formats: [mp3]
+course: thought
```
Comment (buddhist-uni) [inline _content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md:11]: So, here we have another "secular" piece put in a Buddhist folder as a provocation... I could imagine a nice dialogue with students about the poem and "how Buddhist is it?"... But I could also see an argument against its inclusion here.  What do you think?

**Review by buddhist-uni — APPROVED** [review]
Okay, no worries. Besides talking about the format field and markdown syntax was there anything else on the agenda?

**Comment by JJM317** [comment]
Nothing else that I can think of. But, actually,  my appointment was moved
to 8 am, so we'll have to cancel our meeting completely,  sorry. I'm free
Friday morning (US time). Next week I'll be out of town seeing family, so
if we miss this week, it will be another 2 weeks. Up to you.

Kindly,
Joe

**Pull request merged**